src/registry_schemas/example_data/schema_data.py

```diff
@@ -2568,6 +2568,59 @@
     }
 }
 
+CORRECTION_CP_SPECIAL_RESOLUTION = {
+    'filing': {
```

Comment:
Consider using the common header and just creating the additional *correction* item to add on.
That will match the other examples in style better.